### PR TITLE
Remove old style of `+build` tags where possible

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Review by w84thesun — APPROVED** [review]

File: tools/old.go [hunk 1]
```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Keep both old and new styles of build tags.
```
Comment (rumyantseva) [inline tools/old.go:15]: That would be great to explain why!

Comment (AlekSi) [reply tools/old.go:15]: There is an explanation below :)

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**